src/docs/asciidoc/tcp-client.adoc

```diff
@@ -302,11 +302,15 @@ public class Application {
 
 == Connection Pool
 
-By default, the `TCP` client uses an "`elastic`" connection pool. This means that the implementation creates a new channel
-if someone tries to acquire a channel but none is in the pool. Also, there is no limit on the maximum concurrent channels.
+By default, the `TCP` client uses a "`fixed`" connection pool with `500` as a maximum number of the channels and
+`45s` as acquisition timeout.
```

Comment:
Please insert "the" before "acquisition" on this line.

Reply:
done